lib/network.go

```diff
@@ -4774,6 +4775,161 @@ type TransactionSpendingLimit struct {
 	DAOCoinLimitOrderLimitMap map[DAOCoinLimitOrderLimitKey]uint64
 }
 
+func (tsl *TransactionSpendingLimit) ToMetamaskString(params *DeSoParams) string {
```

Comment:
This function is kinda gnarly, but I get why we have to do it. Can we add a comment above this function to explain the "why" here?